src/Symfony/Component/Process/Process.php

```diff
@@ -163,6 +167,23 @@ public function __construct($commandline, string $cwd = null, array $env = null,
         $this->pty = false;
     }
 
+    /**
+     * @param string         $command The command line to pass to the shell of the OS
```

Comment:
I would add a description explaining the "valid" use cases for using this method.